Develop an SQL schema for SOPs

```diff
@@ -113,12 +113,38 @@ CREATE TABLE public."A_TECHNOLOGY_MICROARRAY" (
     "Derived_Array_Data_File" text,
     "Array_Data_Matrix_File" text,
     "Derived_Array_Data_Matrix_File" text,
-    "Array_Design_File" text
+    "Array_Design_File" text,
+    "A_Technology_Microarray_ID" integer NOT NULL
```

Comment:
This internal "ID" does not seem to be consistent with the later `id` name used in the tables that follow.  I would suggest changing the name to `id` to make it consistent.

Reply:
done